tests/acceptance/helpers/sharingHelper.js

```diff
@@ -97,6 +97,7 @@ module.exports = {
         for (const share of shares) {
           found = true
           for (const expectedDetail of expectedDetailsTable.hashes()) {
+            expectedDetail.value = userSettings.replaceInlineCode(expectedDetail.value)
```

Comment:
`sharingHelper` should never have gotten the raw datatable. But. that's for a separate PR.